Owncloud iOs Client/Tabs/FileTab/FilesViewController.m

```diff
@@ -2206,9 +2208,10 @@ - (void) refreshSharedPath{
             if (!isSamlCredentialsError) {
                 
                 //GCD to do things async in background queue
+                [self updateCurrentFileToShowIfNeeded];
```

Comment:
I'm not sure if the comment 

> //GCD to do things async in background queue

 is related with the `updateCurrentFileToShowIfNeeded` because in the previous version this function was called inside the dispatch_async but now is being called outside so I think this is not related with the GCD.